### ISSUE TITLE
Feature request: Forced LFS for selecte4d files extensions check.

### ISSUE THREAD

**Issue by SaraDark** [gitbucket/gitbucket#2851]
OPTION CONFIG: Forced LFS for selected files extensions configured in repository config or not if not necessary. To eliminate potentially mistake configs on client side.

**Comment by takezoe** [comment]
What GitBucket actually can do for that?

For example, generating `.gitattributes`? or/and rejecting non-gitlfs push for selected file types?

**Comment by SaraDark** [comment]
yes exactly :) admin manual configure in gitbucket content of .gitattributes in eg. configuration menu and gitbucket  rejecting non-GitLFS push for selected file types or/and other configured options in .gitattributes.

Optionally cool to add  .gitignore file in gitbucket too :)
